211018_4win_main.py

```diff
@@ -93,9 +86,9 @@ def row_win(player):
     print("start")
     for row in range(6):
         count = 0
-        for index in range(7):
-            print(row, index)
-            if matrix[row][index] == player:
+        for column in range(7):
+            print(row, column)

```

Comment:
Leo's gonna be mad, that you are removing his beloved index :D
But I agree with the changes. They make it more readable.

Reply:
ahhhhhh index!!!!!!!!!!